pom.xml

```diff
@@ -7,8 +7,9 @@
     <groupId>org.sqlflow</groupId>
```

Comment:
Just confusion about the `groupId` field, should it be `sqlfow.org` ?

Reply:
[Java conventions](https://maven.apache.org/guides/mini/guide-naming-conventions.html)